src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -295,7 +295,7 @@ def from_json(cls, json_str):
     'AzureCloud',
     endpoints=CloudEndpoints(
         management='https://management.core.windows.net/',
-        resource_manager='https://management.azure.com/',
+        resource_manager='https://eastus2euap.management.azure.com/',
```

Comment:
This causes CI failure:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=750605&view=logs&j=75453f46-9c6c-5df4-6021-c5646ce3452d&t=4b9b4997-7a06-5274-d89c-915764f2e7f7&l=213892

```
____________________ TestCloud.test_metadata_url_endpoints _____________________
self = <azure.cli.core.tests.test_cloud.TestCloud testMethod=test_metadata_url_endpoints>

    @mock.patch.dict('os.environ', {'ARM_CLOUD_METADATA_URL': 'https://management.azure.com/metadata/endpoints?api-version=2019-05-01'})
    def test_metadata_url_endpoints(self):
        clouds = get_known_clouds(refresh=True)
        for cloud in HARD_CODED_CLOUD_LIST:
            metadata_url_cloud = next(c for c in clouds if c.name == cloud.name)
            for k, v1 in cloud.endpoints.__dict__.items():
                v2 = metadata_url_cloud.endpoints.__dict__[k]
                if v1:
>                   self.assertEqual(v1.strip('/'), v2.strip('/'))
E                   AssertionError: 'https://eastus2euap.management.azure.com' != 'https://management.azure.com'
E                   - https://eastus2euap.management.azure.com
E                   ?         ------------
E                   + https://management.azure.com

src/azure-cli-core/azure/cli/core/tests/test_cloud.py:260: AssertionError
```

This can't be fixed because 

1. There are tests like `test_metadata_url_endpoints` which uses `https://management.azure.com`
2. All yaml recordings are based on `https://management.azure.com`. Calling `https://eastus2euap.management.azure.com` will cause mismatch during playback.